### PR TITLE
Updated the programs to take input from the user

### PR DESCRIPTION
**Pull request by ymaheshwari1**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
To take the input from the user rather than providing the hard-coded input.

### Checklist

- [ Y ] I've read the contribution guidelines.
- [ Y ] I've checked the issue list before deciding what to submit.
- [ Y ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#633

### CODE REVIEW

File: Java/cp/RemoveDuplicateElement.java [hunk 1]
```diff
@@ -1,9 +1,12 @@
+// import the Scanner class
```
Comment (atarax665) [inline Java/cp/RemoveDuplicateElement.java]: No need to add this comment, instead add the description of the problem here.

File: Java/cp/RemoveDuplicateElement.java [hunk 2]
```diff
@@ -67,8 +70,19 @@ static int DuplicateElement(int[] a, int n){
...
+        System.out.print("Enter the length of the array : ");
+        int n = sc.nextInt();
+        
+        int a[] = new int[n];
+        System.out.println("Enter the values for the array elements : ");
+        // taking the values for the array from the user
+        for(int i=0; i<n; i++)
+        {
+            a[i] = sc.nextInt();
+        }
```
Comment (atarax665) [inline Java/cp/RemoveDuplicateElement.java:68]: line 93 should also be removed as the hard coded inputs are removed.

File: Java/cp/RemoveDuplicateElementWithoutExtraSpace.java [hunk 3]
```diff
@@ -1,10 +1,13 @@
+// import the Scanner class
```
Comment (atarax665) [inline Java/cp/RemoveDuplicateElementWithoutExtraSpace.java]: Remove this comment, instead add the description of the problem here.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Java/cp/RemoveDuplicateElement.java (hunk 1 above)
Comment (ymaheshwari1) [reply Java/cp/RemoveDuplicateElement.java]: Okay, I will remove the comment. The description of the problem is already present.
https://github.com/TesseractCoding/NeoAlgo/pull/803/files#diff-0c8550122b2ec1ce669256c8a245d914L3

**Comment by ymaheshwari1** [comment]
@atarax665 Also please add the specific labels to the related issue **CH-20** and other. So that the issue and PR are counted in the HakinCodes

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@ymaheshwari1   please attach a sample input/ output for both the codes in commented form below the code section.

**Review by vatsalkesarwani12 — APPROVED** [review]

**Review by Kajol-Kumari — APPROVED** [review]
LGTM! :+1:

**Pull request merged**